src/main/java/ch/njol/skript/effects/EffKill.java

```diff
@@ -26,44 +26,35 @@
 		"Note: This effect does not set the entity's health to 0 (which causes issues), but damages the entity by 100 times its maximum health."})
 @Examples({"kill the player",
 		"kill all creepers in the player's world",
-		"kill all endermen, witches and bats"})
+		"kill all endermen, witches and bats",
+})
```

Comment:
Why did you add a `,` and spaces at the end? Please keep it consistent for both brackets.

Reply:
I thought I removed it, but I didn't realize I didn't, oopsies